Poker.BE/Poker.BE.Domain/Security/UserManager.cs

```diff
@@ -4,36 +4,42 @@
 using System.Text;
 using System.Threading.Tasks;
 using Poker.BE.Domain.Core;
+using Poker.BE.Domain.Utility.Exceptions;
 
 namespace Poker.BE.Domain.Security
 {
-    public class UserManager
-    {
-        #region Properties
-        protected Dictionary <string, User> UsersDictionary;
+	public class UserManager
```

Comment:
# Poor Implementation 👎 👎 👎 
i found this implementation not related to the UC that it suppose to support. i don't see any `///<summery>` over methods, and lack of correct document formatting.

![image](https://cloud.githubusercontent.com/assets/9889268/26474308/e757df62-41b8-11e7-9a69-27dcee20f239.png)